dubbo-common/src/main/java/org/apache/dubbo/common/utils/StringUtils.java

```diff
@@ -48,7 +48,7 @@
     private static final Pattern INT_PATTERN = Pattern.compile("^\\d+$");
     private static final int PAD_LIMIT = 8192;
 
-    private StringUtils() {
+    protected StringUtils() {

```

Comment:
I will revert to older version.  Because some extension class used com.alibaba.dubbo.common.utils.StringUtils, I want write a class Name com.alibaba.dubbo.common.utils.StringUtils and extend the StringUtils.  Finally I found it can not fix issue.